README.md

```diff
@@ -102,6 +102,9 @@ Options:
   If `false` the request won't be sent until all preceeding
   requests in the pipeline has completed.
   Default: `true` if `method` is `HEAD` or `GET`.
+* `onInfo(statusCode, headers)`, function invoked for informational
+  1xx responses. `100 Continue` is ignored and will not invoke `onInfo`.
```

Comment:
I wonder if we only should support known 1xx responses with defined semantics?

i.e.

101
102
103

e.g. 100 is actually not useful for the user (I think?) and is something that would have to be implemented internally.

Reply:
IMO let's go for all 1XX. Custom informational status codes may be implemented e.g. by CloudFlare in the future like they did with 5XX.